packages/traefik-dynamic/templates/traefik.toml

```diff
@@ -0,0 +1,52 @@
+# global config
+port = "{{or (key "config/traefik/port") ":80"}}"
```

Comment:
It looks like there might be a missing end backet here, Github is highlighting a lot of red.